spec.html

```diff
@@ -8113,6 +8113,20 @@
         </emu-alg>
       </emu-clause>
     </emu-clause>
+
+    <emu-clause id="sec-immutable-prototype-exotic-objects">
+      <h1>Immutable Prototype Exotic Objects</h1>
+      <p>An <dfn>immutable prototype exotic object</dfn> is an exotic object that disallows mutation of its prototype.</p>
+
+      <emu-clause id="sec-immutable-prototype-exotic-objects-setprototypeof-v">
+        <h1>[[SetPrototypeOf]] (_V_)</h1>
+        <p>When the [[SetPrototypeOf]] internal method of a immutable prototype exotic object _O_ is called with argument _V_ the following steps are taken:</p>
+        <emu-alg>
+          1. Assert: Either Type(_V_) is Object or Type(_V_) is Null.
+          1. Return *false*.
```

Comment:
I agree with this. Unless I hear objections in the next few minutes I will spec it!